website/static/js/folderPickerNodeConfig.js

```diff
@@ -509,7 +509,20 @@ var FolderPickerViewModel = oop.defclass({
                     custom : FolderPicker.selectView
                 }
             ];
-            }
+            },
+            xhrconfig: $osf.setXHRAuthorization,
+            lazyLoadPreprocess: function(data) {
```

Comment:
Since this is OSF user-facing, it should be called out in changes so QA knows to check for it, yes?